infrastructure/scripts/execution-plan/create-matrix.sh

```diff
@@ -64,6 +64,21 @@ function createMatrix() {
   fi
   debug "grep for pkg: ${grepOutput}"
 
+  # if we have a change in go.mod, we need to rebuild the builder image:
```

Comment:
this is okay, but the block above should be gone.
We don't need to rebuild the builder just because api.proto changed, right?